Formula/libraw.rb

```diff
@@ -24,7 +25,10 @@ class Libraw < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}",
-                          "--disable-dependency-tracking"
+                          "--disable-dependency-tracking",
+                          "ac_cv_prog_c_openmp=-Xpreprocessor\\ -fopenmp",
```

Comment:
- The `\\` is, I think, unnecessary.
- Do we really need OpenMP support? It adds one more dependency